ams/server/pom.xml

```diff
@@ -42,7 +42,7 @@
         <mysql-jdbc.version>8.0.30</mysql-jdbc.version>
         <derby-jdbc.version>10.13.1.1</derby-jdbc.version>
         <commons-dbcp2.version>2.9.0</commons-dbcp2.version>
-        <snakeyaml.version>1.30</snakeyaml.version>
+        <snakeyaml.version>1.33</snakeyaml.version>
```

Comment:
Since both Optimizer and AMS modules have this dependency, should their version be included in dependenciesManagements ?